homeassistant/components/overkiz/climate.py

```diff
@@ -28,6 +32,18 @@ async def async_setup_entry(
         if device.widget in WIDGET_TO_CLIMATE_ENTITY
     )
 
+    # Match devices based on the widget and controllableName
+    # This is for example used for Atlantic APC, where devices with different functionality share the same uiClass and widget.
+    async_add_entities(
```

Comment:
Please collect all entities and make a single call to `async_add_entities` instead.